ajax/libs/cell/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "cell",
+  "filename": "cell.min.js",
+  "description": "A self-constructing web app framework powered by a self-driving DOM",
+  "homepage": "https://www.celljs.org",
+  "license": "MIT",
+  "keywords": ["self-driving", "webapp", "framework"],
+    "repository": {
```

Comment:
Let's delete two space in front of `"reposittory"`, to align as other fields' format.